src/schema.toml

```diff
@@ -53,6 +54,9 @@ images.company_logo = { category = "images", type = "file", name = "Company Logo
 images.home_background = { category = "images", type = "file", name = "Home background", description = "Background image on the home page." }
 
 images.editor_logo = { category = "images", type = "file", internal = "true" }
+images.landing_rooms_thumb = { category = "images", type = "file", internal = "true" }
+images.landing_communicate_thumb = { category = "images", type = "file", internal = "true" }
+images.landing_media_thumb = { category = "images", type = "file", internal = "true" }
```

Comment:
Should people be able to customize the title/text as well? We now have a "longstring" type. 

If not in this PR, I'd create an issue for community members to vote if they want to customize the text/titles on HC for the 3 panel homepage.

Reply:
It's a good thought, but this particular PR is specifically meant for HMC, which is why it's disabled by default and the flags are internal.
I think it's a good idea to consider enabling something like this for HC, but that's a bit of a bigger discussion around the whole redesign and general site customization that we're intentionally side-stepping in this PR.